docs/adding-a-resource.md

```diff
@@ -0,0 +1,119 @@
+# Adding a new resource
+
+All resources are written using Markdown. Markdown files are stored in the
```

Comment:
Should we specify here that there may be HTML on the page for any boxes or more complicated parts on the page?